src/ducks/subject.js

```diff
@@ -14,7 +14,7 @@ const FETCH_SUBJECT_SUCCESS = 'FETCH_SUBJECT_SUCCESS';
 const FETCH_SUBJECT_ERROR = 'FETCH_SUBJECT_ERROR';
 const TOGGLE_FAVORITE = 'TOGGLE_FAVORITE';
 
-const TEMPORARY_HARDCODED_WORKFLOW_ID = '3022';
+const TEMPORARY_HARDCODED_WORKFLOW_ID = '3017';
```

Comment:
As noted in another comment, this hardcoded value needs to be moved to a config file soon-ish, before we forget.